web/templ/pages/inventory-classifications.templ

```diff
@@ -43,150 +43,158 @@ templ InventoryClassificationContent(params InventoryClassificationParams) {
 templ InventoryClassificationTable(params InventoryClassificationParams) {
 	<table role="table" id="inventory-table">
 		<thead role="rowgroup" hx-boost="true" hx-target="#inventory-table" hx-swap="outerHTML">
-			<tr role="row" class="[&>th]:py-2 [&>th]:pr-4 [&>th:not(.text-left)]:pl-4">
-				@SortableColumnHeaderLeft(params, "name", "Name")
-				<th role="columnheader" scope="col" id="image"></th>
-				if params.Stats.HasAttack {
-					@SortableColumnHeader(params, "attack", "Attack")
+			@inventoryTableHeader(params)
+		</thead>
+		<tbody role="rowgroup" class="[&>tr:nth-child(odd)]:bg-gray-100 dark:[&>tr:nth-child(odd)]:bg-zinc-800" hx-boost="true" hx-target="#page-content" hx-swap="innerHTML">
+			for _, inventory := range params.Inventories {
+				@inventoryTableRow(inventory, params.Stats)
+			}
+		</tbody>
```

Comment:
I decided to break out the table header and data rows into separate functions so this parent one can be smaller.